actions/actions_decision.go

```diff
@@ -0,0 +1,180 @@
+package actions
+
+import "strconv"
+
+/*
+type allDecision
+Schema {
+	"command name" : {
+		validation length : execution function
+	}
+}
+*/
+type allDecision map[string]map[int]func() string
```

Comment:
we need better name than allDecisions

Reply:
CommandsExecuter 
CommandsMapper
Or let me know your opinion